src/gei/Commands/MigrateArchiveRepoCommand.cs

```diff
@@ -0,0 +1,186 @@
+            };
+            var githubSourceOrg = new Option<string>("--github-source-org")
+            {
+                IsRequired = true
+            };
+            var sourceRepo = new Option<string>("--source-repo")
+            {
+                IsRequired = true
+            };
+            var githubTargetOrg = new Option<string>("--github-target-org")
+            {
+                IsRequired = true,
+                Description = "Uses GH_PAT env variable."
+            };
+            var targetRepo = new Option<string>("--target-repo")
+            {
+                IsRequired = false,
+                Description = "Defaults to the name of source-repo"
+            };
+            var azureStorageConnectionString = new Option<string>("--azure-storage-connection-string")
+            {
+                IsRequired = true,
+                Description = "The connection string for the Azure storage account. For example: DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=mykey;EndpointSuffix=core.windows.net"
+            };
+            var noSslVerify = new Option("--no-ssl-verify")
+            {
+                IsRequired = false
+            };
+            var verbose = new Option("--verbose")
+            {
+                IsRequired = false
+            };
+
+            AddOption(ghesApiUrl);
+            AddOption(githubSourceOrg);
+            AddOption(sourceRepo);
+            AddOption(githubTargetOrg);
+            AddOption(targetRepo);
+            AddOption(azureStorageConnectionString);
+            AddOption(noSslVerify);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, string, string, string, bool, bool>(Invoke);
+        }
+
+        public async Task Invoke(string ghesApiUrl, string githubSourceOrg, string sourceRepo, string githubTargetOrg, string targetRepo, string azureStorageConnectionString, bool noSslVerify = false, bool verbose = false)
+        {
+            _log.Verbose = verbose;
+
+            // Log all the parameters, except for the connection string, which is a secret
+            _log.LogInformation("Starting Migration Archives...");
+            _log.LogInformation($"GHES SOURCE ORG: {githubSourceOrg}");
+            _log.LogInformation($"GHES SOURCE REPO: {sourceRepo}");
+            _log.LogInformation($"GITHUB TARGET ORG: {githubTargetOrg}");
+            _log.LogInformation($"GHES API URL: {ghesApiUrl}");
+
+            var dateTimeNow = DateTime.Now;
+
+            if (string.IsNullOrWhiteSpace(targetRepo))
+            {
+                _log.LogInformation($"Target repo name not provided, defaulting to same as source repo ({sourceRepo}) + timestamp");
+                targetRepo = $"{sourceRepo}-{dateTimeNow:MMdd-HHmm}";
+            }
+
+            _log.LogInformation($"TARGET REPO: {targetRepo}");
+
+            if (string.IsNullOrWhiteSpace(azureStorageConnectionString))
```

Comment:
this arg is set as required, so it can never be empty

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/18c42213ef0434e27bbfe45424c12cb311af7d5f